frontend/src/components/form/error-info-field.tsx

```diff
@@ -0,0 +1,31 @@
+import { FormErrorMessage, ListItem, Text, UnorderedList } from '@redpanda-data/ui';
+import type { AnyFieldApi } from '@tanstack/react-form';
+
+interface ErrorInfoFieldProps {
+  field: AnyFieldApi;
+  index?: string;
+}
+
+/**
+ * This component needs to be used inside a FormControl component.
+ * It will display all error messages in a list format for a given form field.
+ * The form field needs the `isInvalid` prop set to true and it had to be touched by the user.
+ * @see https://v2.chakra-ui.com/docs/components/form-control#error-message
+ */
+export const ErrorInfoField = ({ field, index }: ErrorInfoFieldProps) => {
+  if (field.state.meta.errors?.length === 0) {
+    return null;
+  }
+
+  return (
+    <FormErrorMessage>
+      <UnorderedList>
+        {field.state.meta.errors.map((error) => (
+          <ListItem key={index ?? error.message}>
```

Comment:
If index is undefined, using error.message as a key might not be reliable (if messages are repeated). I'd try to use some unique id. Also I don't like that it's coming as a prop, and I use it as a key for `errors.map`

Reply:
Fixed